src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -158,7 +161,12 @@ export default defineComponent({
     newClientId() {
       return Object.keys(this.modelValue).length + 1;
     },
+
+    atLeastOneCheckedError() {
```

Comment:
can rename to `atLeastOneChecked` and you can add return type which is nice:

`atLeastOneChecked(): boolean {...`